kubernetes/docs/try-with-minikube.md

```diff
@@ -77,7 +77,7 @@ npm run docker:build
 
 ```sh
 helm dependency build kubernetes/shopping-app
-helm install --name shopping-app --debug kubernetes/shopping-app
+helm upgrade --install shopping-app --debug kubernetes/shopping-app -n shopping-app --create-namespace
```

Comment:
```suggestion
helm upgrade --install shopping-app --debug kubernetes/shopping-app
```

Reply:
agreed - thanks for the review.
but removing --name
and using upgrade --install
makes sense, right?

Reply:
removing `--name` definitly makes sense as it does not work with helm v3, not sure about `upgrade --install`, I personally prefer using it but someone new to helm might not understand what it does without reading the docs/help.